doc/api/process.markdown

```diff
@@ -16,6 +16,34 @@ continue.
 as [`process.exit()`][] or uncaught exceptions, and should not be used as an
 alternative to the `'exit'` event unless the intention is to schedule more work.
 
+## Event: 'deprecation'
```

Comment:
I'd be fine with that also. I originally had deprecations emitted as
warnings and only split this out following the feedback from Doug. That can
be reverted easily, however.
On Mar 10, 2016 8:41 PM, "Jeremiah Senkpiel" <notifications@github.com>
wrote:

> In doc/api/process.markdown
> <https://github.com/nodejs/node/pull/4782#discussion_r55787034>:
>
> > @@ -16,6 +16,34 @@ continue.
> >  as [`process.exit()`][] or uncaught exceptions, and should not be used as an
> >  alternative to the `'exit'` event unless the intention is to schedule more work.
> >
> > +## Event: 'deprecation'
>
> Hmmm, if deprecations were still also emitted on warning and we moved to
> deprecate deprecate I would be ok with that? Currently it's a little
> split for not much reason. Maybe that's what has existed outside but I
> don't see how it will be goof for our users down the line.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nodejs/node/pull/4782/files#r55787034>.
>